src/test/miner_tests.cpp

```diff
@@ -284,7 +284,7 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity) {
     // Note that by default, these tests run with size accounting enabled.
     GlobalConfig config;
     // Don't check this functionality in this test.
-    config.SetEnableMinerFund(false);
+    config.SetEnableMinerFund(true);
```

Comment:
What's this?

Reply:
Setting it to false didn't do anything before, now it does so it needs to be turned back on.